packages/mds-compliance-engine/engine/helpers.ts

```diff
@@ -214,29 +194,16 @@ export function getProviderIDs(provider_ids: UUID[] | undefined | null) {
 }
 
 /**
- * The rule matches this event if a transient event_type and possible resultant states,
- * or the final event_type and explicitly encoded vehicle_state match with the rule's status definitions.
- * e.g. if the rule states are { reserved: [] } and there's an event with event_types
- *      [trip_end, reservation_start, trip_start], there's an implication
- *      that the vehicle entered the reserved state after reservation_start,
- *      even if the final state of the event is on_trip, and the rule will match.
+ * Suppose we have an event with event_types `["trip_end", "battery_low"]`, and a
+ * `vehicle_state` of `available`, and a rule on a policy
+ * where the `states` are `{ "available": ["trip_end"] }`. That should be a match.
```

Comment:
`['trip_end', 'battery_low']` would result in a state of `non_operational` instead of `available`, right?

Reply:
Ok I'll invent an example using a valid event that still demonstrates the new logic